TRAIN_AND_RUN_README.md

```diff
@@ -1,61 +1,132 @@
 # Step-by-step how to run the Angel System pipeline
 
 ## Table of Contents
-- [Local installation](#localinstallation)
-- [Docker installation](#dockerinstallation)
+- [Local installation](#local-installation)
+- [Docker installation](#docker-installation)
 - [Data and pretrained models](#data)
-- [Training](#training)
+- [Training](#training-procedure)
 - [Training on lab data](#lab_data)
-- [Docker local testing with pre-recorded data](#local)
-- [Real-time](#realtime)
+- [Docker local testing with pre-recorded data](#docker-local-testing)
+- [Real-time](#docker-real-time)
 
-## Local Installation <a name = "localinstallation"></a>
+## Local Installation
 
 Follow the following steps (the optional steps are for active development purposes):
 
 ##### Get required repositories:
 ```
-(optional) git clone git@github.com:PTG-Kitware/TCN_HPL.git
-(optional) git clone git@github.com:PTG-Kitware/yolov7.git
 git clone git@github.com:PTG-Kitware/angel_system.git
 cd angel_system
 git submodule update --init --recursive
 ```
 
 ##### Create the environment
 ```
+# IF YOU DON'T ALREADY HAVE PYTHON 3.8.10 AVAILABLE
 conda create --name angel_systen python=3.8.10
 conda activate angel_test_env
-poetry lock --no-update
 poetry install
-```
 
-##### 
+# OR JUST
+poetry install
+```
 
-## Docker Installation <a name = "dockerinstallation"></a>
+## Docker Installation
 
 Follow the following steps (the optional steps are for active development purposes):
-##### Get required repositories:
+### Get required repositories:
 ```
-(optional) git clone git@github.com:PTG-Kitware/TCN_HPL.git
-(optional) git clone git@github.com:PTG-Kitware/yolov7.git
 git clone git@github.com:PTG-Kitware/angel_system.git
 cd angel_system
 git submodule update --init --recursive
 ```
-##### Create the environment
+### Create the environment
 ```
 ./angel-docker-build.sh -f
 ./angel-workspace-shell.sh
 ```
 
-##### Inside the docker container:
+### Inside the docker container:
 ```
 ./workspace_build.sh; source install/setup.sh
 ```
 
-## Data <a name = "data"></a>
-- On gyges, raw data is located at `/data/PTG/medical/bbn_data/Release_v0.5/v0.56/<task>`
+## Data
+
+### Object Detection Source Data
+TODO
+
+### Pose Detection Source Data
+TODO
+
+### TCN Source Data
+
+#### BBN Medical Datasets
+Source data from BBN can be acquired from https://bbn.com/private/ptg-magic/.
+Consult a team member for login information.
+
+Each task ("skill") has their own sub-page ("In-Lab Data" link) from which sets
+of data are described and referred to.
+Data is stored on their SFTP server, to which the "Click to Download" links
+refer to.
+
+Storage of downloaded ZIP archives, and their subsequent extractions, should
+follow the pattern.
+A script is provided 
```

Comment:
recommend listing the location of the script for easy reference

Reply:
I see I forgot to finish that sentence.